### PR TITLE
Enable OpenVINO export of loaded model

### PR DESCRIPTION
**Pull request by echarlaix**
```python
from diffusers import StableDiffusionPipeline
from optimum.intel import OVStableDiffusionPipeline
from optimum.exporters.openvino import export_from_model

model_id = "runwayml/stable-diffusion-v1-5"
model = StableDiffusionPipeline.from_pretrained(model_id)

save_dir = "ov_model"
export_from_model(model, output=save_dir, task="stable-diffusion")
pipeline = OVStableDiffusionPipeline.from_pretrained(save_dir)
images = pipeline("sailing ship in storm by Rembrandt").images
```

### CODE REVIEW

File: .github/workflows/test_openvino.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ jobs:
         python -m pip install --upgrade pip
         # install PyTorch CPU version to avoid installing CUDA packages on GitHub runner without GPU
         pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
+        pip install git+https://github.com/huggingface/optimum.git
```
Comment (echarlaix) [inline .github/workflows/test_openvino.yml:35]: TODO : remove

File: examples/openvino/stable-diffusion/train_text_to_image_qat.py [hunk 2]
```diff
@@ -700,7 +650,7 @@ def get_nncf_config(pipeline, dataloader, args):
                 "ignored_scopes": [
                     "{re}.*__add___[0-2]",
                     "{re}.*layer_norm_0",
-                    "{re}.*Attention.*/bmm_0",
+                    # "{re}.*Attention.*/bmm_0",
```
Comment (echarlaix) [inline examples/openvino/stable-diffusion/train_text_to_image_qat.py:653]: get the following error from nncf when left uncommented : 
`No match has been found among the model operations for the following ignored/target scope definitions`

File: tests/openvino/test_modeling.py [hunk 3]
```diff
@@ -494,6 +496,10 @@ class OVModelForCausalLMIntegrationTest(unittest.TestCase):
     @parameterized.expand(SUPPORTED_ARCHITECTURES)
     def test_compare_to_transformers(self, model_arch):
         model_id = MODEL_NAMES[model_arch]
+
+        if "gptq" in model_arch:
+            self.skipTest("GPTQ model loading unsupported with AutoModelForCausalLM")
```
[the remark this answers is not in the document]
Comment (eaidova) [reply tests/openvino/test_modeling.py:501]: it still not solve fully inference issue, but probably will allow loading gptq models with export_from_model

[the remark this answers is not in the document]
Comment (eaidova) [reply tests/openvino/test_modeling.py:501]: BTW, in our internal openvino tests opt125-gptq works on torch cpu, only required to add torch_dtype=torch.float32 into from_pretrained for that

[the remark this answers is not in the document]
Comment (echarlaix) [reply tests/openvino/test_modeling.py:501]: this makes sense, just might make more sense to have it in optimum directly (not directly related to openvino), can be added in a following PR!

**Pull request merged**